OWNERS_ALIASES

```diff
@@ -5,9 +5,9 @@ aliases:
     - luxas
     - roberthbailey
```

Comment:
Given that robert is moving on could you also remove him from the OWNERS files?

Reply:
I'll remove him. Thanks for letting me know!